<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaveformWidget</class>
 <widget class="QWidget" name="WaveformWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>435</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Period:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblSize">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lblFinal">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QCustomPlot" name="plot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="txtPeriod"/>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="txtSize"/>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="txtFinal"/>
   </item>
   <item row="5" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnPlot">
       <property name="text">
        <string>Plot Waveform</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/resources/images/plot.png</normaloff>:/resources/images/plot.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>17</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Download to File</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/resources/images/download.png</normaloff>:/resources/images/download.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>17</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpload">
       <property name="text">
        <string>Upload to Flash</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/resources/images/upload.png</normaloff>:/resources/images/upload.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>17</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Final:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lblPeriod">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Delay:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="lblDelay">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="txtDelay"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>view/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtSize</tabstop>
  <tabstop>txtPeriod</tabstop>
  <tabstop>txtFinal</tabstop>
  <tabstop>btnPlot</tabstop>
  <tabstop>btnUpload</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
